BlogTemplate/Pages/Post.cshtml.cs

```diff
@@ -41,18 +41,30 @@ public IActionResult OnPostPublishComment([FromRoute] int id)
         {
             Post = _dataStore.GetPost(id);
 
+            Comment comment = new Comment

```

Comment:
this could go inside the ModelState.IsValid block, since it's only needed once the form has been validated